content/docs/rkvst-basics/creating-an-asset/index.md

```diff
@@ -18,7 +18,9 @@ aliases:
 
 An Asset can be anything: a Connected Machine, a Shipping Container, or even a Data Set. It can be any physical or digital object with an associated Name, Description, and Attributes.
 
-Each Asset will have a history of any actions performed upon it by any actor. 
+Each Asset will have a history of any actions performed upon it by any actor.
+
+You may wish to publicly attest an Asset's history. For more information on how to do so, see [Public Attestation](https://docs.rkvst.com/docs/beyond-the-basics/public-attestation/).
```

Comment:
Minor: I (as a non-docs person) might rephrase this as: "RKVST enables you to publicly attest to the provenance of your Assets. To learn how, see {Link}"

Reply:
I prefer that wording as well, I changed the paragraph to note the options for permissioned sharing or public attestation.